src/watch/hearRingiOSWatchOS/Shared/PhoneWatchConnectivity.swift

```diff
@@ -163,7 +163,7 @@ extension Connectivity: WCSessionDelegate {
     ) {
         update(from: message)
 
-        let key = "alarmEnabled"
+        let key = "SettingsOrAlarm"
```

Comment:
Why's this being changed in this branch? I'm okay with letting this go through I just wanna know the reason and that this isn't a lingering thing that might mess something up.